numpy/testing/tests/test_utils.py

```diff
@@ -432,6 +437,109 @@ def test_nan_items(self):
                 lambda: self._assert_func(ainf, anan))
 
 
+class TestArrayAssertLess(unittest.TestCase):
+
+    def setUp(self):
+        self._assert_func = assert_array_less
```

Comment:
Not much reason for this unless there is some form of inheritance somewhere, but if you like it...